implementations/rust/ockam/ockam/src/credential/holder.rs

```diff
@@ -107,7 +107,7 @@ impl CredentialHolder {
         proof_request_id: [u8; 32],
     ) -> Result<Vec<CredentialPresentation>, CredentialError> {
         // To prove the id-secret is the same across credentials we use a Schnorr proof
-        // which requires that the proof blinding factor be the same. If there's only one credential
+        // which requires that the proof blinding proof be the same. If there's only one credential
```

Comment:
This comment change is not correct. It was correct before. Is this accidental?

Reply:
I was doing find&replace. Good catch. Will fix

Reply:
Fixed